src/ui/components/DataTable/Controls.tsx

```diff
@@ -16,6 +16,7 @@ const style = {
 
 const Controls = ({
 	enableFilters,
+	enableCustomize,
```

Comment:
Rename to `enableCustomizeColumns` just to be a little more clear.

Reply:
renamed